app/Console/Commands/NetworkData/ProcessNetworkData.php

```diff
@@ -178,20 +175,20 @@ private function processPilots()
             $flight = Pilot::firstOrNew([
                 'account_id'        => $account->id,
                 'callsign'          => $pilotData['callsign'],
-                'flight_type'       => $pilotData['planned_flighttype'],
-                'departure_airport' => $pilotData['planned_depairport'],
-                'arrival_airport'   => $pilotData['planned_destairport'],
-                'connected_at'      => Carbon::createFromFormat('YmdHis', $pilotData['time_logon']),
+                'flight_type'       => $pilotData['flight_plan']['flight_rules'],
+                'departure_airport' => $pilotData['flight_plan']['departure'],
+                'arrival_airport'   => $pilotData['flight_plan']['arrival'],
+                'connected_at'      => Carbon::create($pilotData['logon_time']),
                 'disconnected_at'   => null,
             ]);
 
             $flight->fill([
-                'alternative_airport' => $pilotData['planned_altairport'],
-                'aircraft'            => $pilotData['planned_aircraft'],
-                'cruise_altitude'     => $pilotData['planned_altitude'],
-                'cruise_tas'          => $pilotData['planned_tascruise'],
-                'route'               => $pilotData['planned_route'],
-                'remarks'             => $pilotData['planned_remarks'],
+                'alternative_airport' => $pilotData['flight_plan']['alternate'],
+                'aircraft'            => $pilotData['flight_plan']['aircraft'],
+                'cruise_altitude'     => $pilotData['flight_plan']['altitude'],
+                'cruise_tas'          => $pilotData['flight_plan']['cruise_tas'],
+                'route'               => $pilotData['flight_plan']['route'],
+                'remarks'             => $pilotData['flight_plan']['remarks'],
```

Comment:
Maybe worth defining a 'getFlightplanField' method instead of getting the flight plan key each time?

Reply:
I think the extraction could be overkill - my goal is to remove the need for this entirely and start using the API to grab the stats we need rather than storing it ourselves